pnpm-lock.yaml

```diff
@@ -4273,7 +4692,7 @@ packages:
       - rollup
     dev: true
 
-  /vite/4.1.1:
+  /vite@4.1.1:
```

Comment:
would be nice to use 4.2.1 everywhere so that we don't have multiple versions

Reply:
Other one is site-kit, which is better to not touch IMO. Let's leave it like this for a while IG? Then if we move site-kit back here, it should not be an issue